javaparser-core-testing/src/test/java/com/github/javaparser/javadoc/JavadocTest.java

```diff
@@ -89,7 +90,8 @@ void descriptionAndBlockTagsAreRetrievable() {
     @Test
     void inlineTagsAreParsable() {
         String docText =
-                "Returns the {@link TOFilename}s of all files that existed during the requested" + EOL +
+                "{@inheritDoc}" + EOL +
```

Comment:
Why changing the test instead od adding a new one? Isn't this one different?

Reply:
This test is supposed to check inline tags. I can split it up?